external/binplace/binplace.depproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please make the name of this more descriptive.  Either call it "binplacePackages" or combine it with harvestPackages.